_pages/platform/organization-profile.md

```diff
@@ -50,7 +50,7 @@ Once your organization has been created you can edit all the provided informatio
 
 ### Inviting users to an organization
 
-You can invite users to your organization by clicking the **Invite Member** button at the upper-right corner of the organization profile. You can invite members either by using their Algorithmia account name or by providing their email address. Users invited by account name will receive an email and as well as an in-platform notification to join the organization.
+You can invite users to your organization by clicking the **Invite Member** button at the upper-right corner of the organization profile. You can invite members either by using their Algorithmia account name or by providing their email addresses. Users invited by account name will receive an email and as well as an in-platform notification to join the organization.
```

Comment:
```suggestion
You can invite users to your organization by clicking the **Invite Member** button at the upper-right corner of the organization profile. You can invite a member either by using their Algorithmia account name or by providing their email address. Users invited by account name will receive an email and as well as an in-platform notification to join the organization.
```
Thanks; the use of "their" was intended to be singular and gender neutral, but I see it wasn't clear. I've updated the wording to be clearer, how does this sound?